appveyor.yml

```diff
@@ -3,7 +3,10 @@
 # Test against these versions of Node.js.
 environment:
   matrix:
+    - nodejs_version: "6"
+    - nodejs_version: "8"
     - nodejs_version: "10"
+    - nodejs_version: "11"
```

Comment:
do we need to test all of the versions on windows? I think we only tested latest due to CI speed reasons

Reply:
Well. At least 10 and 11 so we can try with and without WorkerThreads